src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { DrawerProps, Minors } from './Drawer.types';
+import { AnimatePresence, motion } from 'framer-motion';
+import { withIris } from '../../utils';
+import { DrawerContainer } from './Minors/DrawerContainer';
+import { getAnimation } from './util';
+
+const DrawerComponent = ({
+  placement,
+  isOpen,
+  size = 'md',
+  // autoFocus,
+  // initialFocusRef,
+  // finalFocusRef,
+  // returnFocusOnClose,
+  // preserveScrollBarGap,
+  children,
+}: DrawerProps) => {
+  return (
+    <AnimatePresence>
```

Comment:
We can give this an `exitBeforeEnter` prop so the exit animation can happen before framer removes the element from the dom.